api/src/main/java/org/pmiops/workbench/api/CloudTaskInitialCreditsExpiryController.java

```diff
@@ -62,7 +62,8 @@ public class CloudTaskInitialCreditsExpiryController
   }
 
   @SuppressWarnings("unchecked")
-  public ResponseEntity<Void> handleInitialCreditsExpiry(
```

Comment:
because of this incorrect name, the Delegate's `handleInitialCreditsExpiryBatch()` was called instead, which returns 501 Not Implemented